src/parsing/Parse_rule.ml

```diff
@@ -1669,6 +1705,8 @@ let parse_mode env mode_opt (rule_dict : dict) : R.mode =
       in
       `Extract
         { formula; dst_lang; extract_rule_ids; extract; reduce; transform }
+  | Some ("semgrep_internal_postprocessor", _), _ ->
```

Comment:
please use a better name, maybe semgrep_internal_secrets.
Otherwise I have no idea wiaht postprocessor mean/

Reply:
This was a compromise with Lewis to make him ok with merging the semgrep changes into the public facing repo; it's intentionally vague.

Reply:
@yosefAlsuhaibani We should add a TODO to make this more specific

Reply:
But we have lots of use of `Secrets in the code ... in the public repo ... even this discussion is in the public repo
so I don't really get obfuscating it in the rule.

Reply:
The rules are user facing, so I believe the goal is to keep users from getting used to `mode: secrets` (since this is experimental and we think it will change) and to keep it a little quieter, since people are more likely to see rules than look at our code base. I don't think we're trying that hard to keep it secret though, all our paying customers know it's coming soon and it's not like they're under NDA.

Lewis would prefer we also don't use secrets in the code but he's ok with it if it would be very annoying for us as maintainers.